### ISSUE TITLE
Support Django-CMS 3.3.0 ?

### ISSUE THREAD

**Issue by bartmika** [nephila/djangocms-multisite#6]
I've tried installing Django-CMS 3.3.0 I keep getting errors. One of which is:

``` python
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/bartlomiejmika/Developer/cloudmasterstudios/djangocms-cloudmasterstudios/env/lib/python3.4/site-packages/django/core/management/__init__.py", line 353, in execute_from_command_line
    utility.execute()
  File "/Users/bartlomiejmika/Developer/cloudmasterstudios/djangocms-cloudmasterstudios/env/lib/python3.4/site-packages/django/core/management/__init__.py", line 327, in execute
    django.setup()
  File "/Users/bartlomiejmika/Developer/cloudmasterstudios/djangocms-cloudmasterstudios/env/lib/python3.4/site-packages/django/__init__.py", line 18, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/Users/bartlomiejmika/Developer/cloudmasterstudios/djangocms-cloudmasterstudios/env/lib/python3.4/site-packages/django/apps/registry.py", line 108, in populate
    app_config.import_models(all_models)
  File "/Users/bartlomiejmika/Developer/cloudmasterstudios/djangocms-cloudmasterstudios/env/lib/python3.4/site-packages/django/apps/config.py", line 202, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/local/Cellar/python3/3.4.2_1/Frameworks/Python.framework/Versions/3.4/lib/python3.4/importlib/__init__.py", line 109, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 2254, in _gcd_import
  File "<frozen importlib._bootstrap>", line 2237, in _find_and_load
  File "<frozen importlib._bootstrap>", line 2226, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1200, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1129, in _exec
  File "<frozen importlib._bootstrap>", line 1471, in exec_module
  File "<frozen importlib._bootstrap>", line 321, in _call_with_frames_removed
  File "/Users/bartlomiejmika/Developer/cloudmasterstudios/djangocms-cloudmasterstudios/env/lib/python3.4/site-packages/cms/models/__init__.py", line 3, in <module>
    from .pagemodel import *  # nopyflakes
  File "/Users/bartlomiejmika/Developer/cloudmasterstudios/djangocms-cloudmasterstudios/env/lib/python3.4/site-packages/cms/models/pagemodel.py", line 1589, in <module>
    _reversion()
  File "/Users/bartlomiejmika/Developer/cloudmasterstudios/djangocms-cloudmasterstudios/env/lib/python3.4/site-packages/cms/models/pagemodel.py", line 1585, in _reversion
    exclude_fields=exclude_fields
  File "/Users/bartlomiejmika/Developer/cloudmasterstudios/djangocms-cloudmasterstudios/env/lib/python3.4/site-packages/cms/utils/helpers.py", line 37, in reversion_register
    from cms.utils import reversion_hacks
  File "/Users/bartlomiejmika/Developer/cloudmasterstudios/djangocms-cloudmasterstudios/env/lib/python3.4/site-packages/cms/utils/reversion_hacks.py", line 18, in <module>
    from reversion.models import Revision, Version, post_revision_commit  # NOQA  # nopyflakes
ImportError: cannot import name 'post_revision_commit'
(env) Bartlomiejs-Mac-mini:djangocms-cloudmasterstudios 
```

And my **requirements.txt** file is:

``` python
Babel==2.3.4
Django==1.9.7
Django-Select2==4.3.2
Pillow==3.2.0
URLObject==2.4.0
Unidecode==0.04.19
YURL==0.13
aldryn-apphooks-config==0.2.7
aldryn-boilerplates==0.7.4
aldryn-categories==1.0.3
aldryn-common==1.0.3
aldryn-disqus==1.0.0
aldryn-newsblog==1.2.2
aldryn-people==1.2.1
aldryn-reversion==1.0.9
aldryn-search==0.2.12
aldryn-translation-tools==0.2.1
argparse==1.4.0
backport-collections==0.1
beautifulsoup4==4.4.1
boto==2.40.0
bunch==1.0.1
click==6.6
cmsplugin-filer==1.1.0
dj-database-url==0.4.1
django-appconf==1.0.2
django-appdata==0.1.5
django-classy-tags==0.7.2
django-cms==3.3.0rc4
django-compressor==2.0
django-filer==1.2.0
django-formtools==1.0
django-haystack==2.4.1
django-htmlmin==0.9.1
django-jsonfield==1.0.0
django-mailgun==0.9.1
django-meta==1.3
django-meta-mixin==0.3.0
django-mptt==0.8.4
django-multisite==1.2.6
django-parler==1.6.4
django-phonenumber-field==1.1.0
django-polymorphic==0.9.2
django-reversion==2.0.4
django-sekizai==0.9.0
django-sortedm2m==1.3.0
django-spurl==0.6.4
django-standard-form==1.1.1
django-storages==1.4.1
django-taggit==0.20.0
django-taggit-autosuggest==0.3.0
django-taggit-templatetags==0.2.5
django-templatetag-sugar==1.0
django-treebeard==4.0.1
djangocms-admin-style==1.2.2
djangocms-apphook-setup==0.1.2
djangocms-attributes-field==0.0.2
djangocms-blog==0.8.3
djangocms-column==1.6.0
djangocms-googlemap==0.4.0
djangocms-inherit==0.2.1
djangocms-installer==0.8.10
djangocms-link==1.8.0
djangocms-style==1.7.0
djangocms-text-ckeditor==3.0.0
easy-thumbnails==2.3
gunicorn==19.6.0
html5lib==0.9999999
idna==2.1
lxml==3.6.0
ordereddict==1.1
phonenumbers==7.4.3
phonenumberslite==7.4.3
psycopg2==2.6.1
python-dateutil==2.5.3
python-dotenv==0.5.1
python-jsonrpc==0.8.4
python-slugify==1.2.0
pytz==2016.4
rcssmin==1.0.6
requests==2.10.0
requests-file==1.4
rjsmin==1.0.12
six==1.10.0
tldextract==2.0.1
tzlocal==1.2.2
https://github.com/nephila/djangocms-multisite/archive/master.zip
```

When I look at your example, it uses **Django-CMS 3.2.5**. Is it possible to support **3.3.0**?

**Comment by bartmika** [comment]
Another very common problem I get is this:

``` python
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/bartlomiejmika/Developer/cloudmasterstudios/djangocms-cloudmasterstudios/env/lib/python3.4/site-packages/django/core/management/__init__.py", line 354, in execute_from_command_line
    utility.execute()
  File "/Users/bartlomiejmika/Developer/cloudmasterstudios/djangocms-cloudmasterstudios/env/lib/python3.4/site-packages/django/core/management/__init__.py", line 346, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/bartlomiejmika/Developer/cloudmasterstudios/djangocms-cloudmasterstudios/env/lib/python3.4/site-packages/django/core/management/base.py", line 394, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/Users/bartlomiejmika/Developer/cloudmasterstudios/djangocms-cloudmasterstudios/env/lib/python3.4/site-packages/django/core/management/base.py", line 445, in execute
    output = self.handle(*args, **options)
  File "/Users/bartlomiejmika/Developer/cloudmasterstudios/djangocms-cloudmasterstudios/env/lib/python3.4/site-packages/django/core/management/commands/migrate.py", line 179, in handle
    created_models = self.sync_apps(connection, executor.loader.unmigrated_apps)
  File "/Users/bartlomiejmika/Developer/cloudmasterstudios/djangocms-cloudmasterstudios/env/lib/python3.4/site-packages/django/core/management/commands/migrate.py", line 318, in sync_apps
    cursor.execute(statement)
  File "/Users/bartlomiejmika/Developer/cloudmasterstudios/djangocms-cloudmasterstudios/env/lib/python3.4/site-packages/django/db/backends/utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/Users/bartlomiejmika/Developer/cloudmasterstudios/djangocms-cloudmasterstudios/env/lib/python3.4/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/Users/bartlomiejmika/Developer/cloudmasterstudios/djangocms-cloudmasterstudios/env/lib/python3.4/site-packages/django/db/utils.py", line 98, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/Users/bartlomiejmika/Developer/cloudmasterstudios/djangocms-cloudmasterstudios/env/lib/python3.4/site-packages/django/utils/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/Users/bartlomiejmika/Developer/cloudmasterstudios/djangocms-cloudmasterstudios/env/lib/python3.4/site-packages/django/db/backends/utils.py", line 62, in execute
    return self.cursor.execute(sql)
django.db.utils.ProgrammingError: relation "cms_cmsplugin" does not exist
```

**Comment by bartmika** [comment]
This issue is dependent on https://github.com/divio/django-cms/issues/5441 being fixed to this issue.

**Comment by yakky** [comment]
To fix divio/django-cms#5441 you can just pin django-reversion version
djangocms-multisite supports django CMS 3.3 already
